### ISSUE TITLE
Add conventional changelog tool + documentation

### ISSUE THREAD

**Issue by sinedied** [ngx-rocket/generator-ngx-rocket#160]
Base on [this article](https://medium.com/@tomastrajan/6-best-practices-pro-tips-for-angular-cli-better-developer-experience-7b328bc9db81) idea (point 6), I think that support for conventional changelog would be a nice addition and should be proposed as an option or add-on (to be discussed).

Any feedback and propositions welcome here 😃 

It's already used in angular and other popular projects, and is a great way in my opinion to maintain changelog for apps. Here is the commit "spec": https://conventionalcommits.org

Tasks:
- [ ] add doc to explain the basics of conventional commits and syntax (+ references)
- [ ] add a `release` script that call `standard-version` to generate changelog, bump version, create a git tag and eventually push it

**Comment by Fairen** [comment]
To ensure the commit messages are corectly formated for conventional changelog generation, theses tools could be intersting : 
- `conventional-changelog-cli`
- `commitizen` and `it-cz`

What do you think ?

**Comment by sinedied** [comment]
Yes, you can take some inspiration in the config used for publishing this package itself. Though the final public is done with `semantic-release` which may not be applicable to publish an angular app, that's why I suggested `standard-version`.
